### ISSUE TITLE
[meta] (temporarily) Publish updated HTML Ruby details as CSSWG Note or WD

### ISSUE THREAD

**Issue by tabatkins** [w3c/csswg-drafts#6102]
The HTML Ruby spec describes a ruby model that is not fully implemented by any browser, and which mixes layout and markup details together, in a way that doesnt' accord well with the CSS ruby model. We've been intending to fix it for several years, but recently @frivoal and @fantasai [actually wrote out the fixes](https://github.com/whatwg/html/issues/1771).

HTML is a little gun-shy about accepting changes to the set of HTML elements without explicit multi-implementor interest (after the unfortunate debacle that was the `dialog` element's multi-year "oh yeah, sounds great, we'll implement it someday" travails), and right now only Firefox implements the proposed spec, so the HTML editors are loathe to merge the PR at the moment.

Long-standing PRs aren't great (they're bad for discoverability, they tend to bitrot, etc), and non-browser impls are interested in making sure their Ruby impl matches what browsers will end up doing, so they'd like a document to refer to before HTML pulls the trigger and merges things, too.

Just to give it a semi-official place to live (rather than fantasai or Florian's personal githubs, or just living as a PR), would it make sense to have us publish the document as a WG Note or UD, with the intro explicitly saying that this is currently a *proposed* HTML feature and is planned to merge into HTML in the future?

**Comment by svgeesus** [comment]
I agree that having this in a Note is much better (n terms of discoverability, facilitating multi-stakeholder discussion) than having it as an unmerged PR or a personal site.
